src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -97,45 +154,113 @@ export const TxParametersDetail = ({
-              isOutOfOrder={isTxNonceOutOfOrder}
-              color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-            >
-              Safe nonce
-            </ColoredText>
-            <ColoredText
-              size="lg"
-              isOutOfOrder={isTxNonceOutOfOrder}
-              color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-            >
-              {txParameters.safeNonce}
-            </ColoredText>
-          </TxParameterWrapper>
-
-          {showSafeTxGas && (
-            <TxParameterWrapper>
-              <Text
-                size="lg"
-                color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-              >
-                SafeTxGas
-              </Text>
-              <Text
-                size="lg"
-                color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-              >
-                {txParameters.safeTxGas}
-              </Text>
-            </TxParameterWrapper>
-          )}
+          {showSafeTxGas && <TxParameter name="SafeTxGas" value={txParameters.safeTxGas || '0'} color={color} />}
           <StyledButtonLink color="primary" textSize="xl" onClick={onEdit}>
             Edit
           </StyledButtonLink>
+          {storedTx?.length > 0 && <TxAdvancedParametersDetail tx={storedTx[0]} />}
         </AccordionDetailsWrapper>
       </AccordionDetails>
     </Accordion>
   )
 }
+
+const TxAdvancedParametersDetail = ({ tx }: { tx: Transaction }) => {
+  const { txData, detailedExecutionInfo } = tx?.txDetails || {}
+
+  if (!txData || !detailedExecutionInfo) {
+    return null
+  }
+
+  const { value, to, operation, hexData } = txData
+  const { safeTxHash, baseGas, gasPrice, gasToken, refundReceiver, confirmations } =
+    (isMultiSigExecutionDetails(detailedExecutionInfo) && detailedExecutionInfo) || {}
+
+  return (
+    <>
+      <StyledDivider />
+
+      {value && <TxParameter name="value" value={value} />}
+
+      {to?.value && (
+        <TxParameter
+          name="to"
+          value={
+            <PrefixedEthHashInfo
+              textSize="lg"
+              hash={to.value}
+              showCopyBtn
+              explorerUrl={getExplorerInfo(to.value)}
+              shortenHash={8}
+            />
+          }
+        />
+      )}
+
+      {safeTxHash && (
+        <TxParameter
+          name="safeTxHash"
+          value={<EthHashInfo textSize="lg" hash={safeTxHash} showCopyBtn shortenHash={8} />}
+        />
+      )}
+
+      {Object.values(Operation).includes(operation) && (
+        <TxParameter name="Operation" value={`${operation} (${Operation[operation].toLowerCase()})`} />
+      )}
+
+      {baseGas && <TxParameter name="baseGas" value={baseGas} />}
+
+      {gasPrice && <TxParameter name="gasPrice" value={gasPrice} />}
```

Comment:
Looks great now, I would just move this check into the component.
`if (value === '') return null`
It can still remain outside for more complex cases like the `Operation` above.